### PR TITLE
Create command for get infomation of player on game.

### LINKED ISSUE #435: [Command] /info name
(nome) - mostra informações do jogador (level, skills, ip, outros chares conectados, posição, velocidade, frags, warns)

### LINKED ISSUE #440: [Command] /spy
/spy (name) - mostra os itens de um jogador, abrindo uma janela

### PR DESCRIPTION
**Pull request by MarcusSilvaDev**

### CODE REVIEW

File: data/extensions/Spells/Commands/InfoCommand.cs [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            Capacity: {target.TotalCapacity}
+            PremiumTime: {target.PremiumTime}
+            Level: {target.Level}
+            Skills:
+            {target.Skills.Where(item => item.Key != SkillType.Level).Select(Item =>   "   * " + Item.Key + ": " + Item.Value.Level + "\n").Aggregate((a, b) => a + b)}
+            """;
+
+       var window = new ListCommandsCommand.TextWindow(info, target.Location, message);
+       
+       window.WrittenBy = "OpenCoreMMO - SERVER";
```
Comment (alissonpadua) [inline data/extensions/Spells/Commands/InfoCommand.cs]: perhaps would be better to not have this text hard coded. Any chance to add on env ?

File: data/extensions/Spells/Commands/InfoCommand.cs [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+       
+       player.Read(window);
+
+       return true;
+    }
+    
+    private static IItemType CreateItemInfo()
+    {
+        var item = new ItemType();
+        item.SetName("Info Status");
```
Comment (alissonpadua) [inline data/extensions/Spells/Commands/InfoCommand.cs:61]: same here =)

File: data/extensions/Spells/Commands/InfoCommand.cs [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        item.SetClientId(2821);
```
Comment (alissonpadua) [inline data/extensions/Spells/Commands/InfoCommand.cs:62]: I'm not sure what that means, but if we create a Enum to handle that ?

File: data/extensions/Spells/Commands/InfoCommand.cs (hunk 1 above)
Comment (MarcusSilvaDev) [reply data/extensions/Spells/Commands/InfoCommand.cs]: Everything is possible to add on env, but what would be a good name for this variable?

File: data/extensions/Spells/Commands/InfoCommand.cs (hunk 3 above)
Comment (MarcusSilvaDev) [reply data/extensions/Spells/Commands/InfoCommand.cs:62]: This approach is similar to the Command "ListCommandSpell" created for you.  I tried to understand this on code The Baiak, but I didn't have success.

File: src/Standalone/appsettings.Local.json
Comment (caioavidal) [inline src/Standalone/appsettings.Local.json:1]: please remove this file from PR

File: data/extensions/Spells/Commands/InfoCommand.cs (hunk 1 above)
Comment (caioavidal) [reply data/extensions/Spells/Commands/InfoCommand.cs]: Lets catch the server name from appsettings and concat in the string:

image

**Review by caioavidal — APPROVED** [review]

**Pull request merged**